src/ocl-client/api.config.ts

```diff
@@ -0,0 +1,14 @@
+import * as dotenv from 'dotenv';
+import { AxiosRequestConfig } from 'axios';
+
+dotenv.config();
+
+export const apiConfig: AxiosRequestConfig = {
+    baseURL: `${process.env.BASE_URL}`,
+    responseType: 'json',
+    headers:{
+        'Authorization': 'Token 309cfade009ec48fe585f3336309d009986ec74d',
```

Comment:
You hard coded your api key in here. Please make sure secrets come from the env as hard coding values creates vulnerabilities

Reply:
I am not sure I wrote that but I'll do so right away.